client/signup/config/step-components.js

```diff
@@ -16,6 +16,12 @@ import SurveyStepComponent from 'signup/steps/survey';
 import ThemeSelectionComponent from 'signup/steps/theme-selection';
 import UserSignupComponent from 'signup/steps/user';
 import PlansStepWithoutFreePlan from 'signup/steps/plans-without-free';
+import JPOSiteTitleComponent from 'signup/steps/jpo-site-title';
```

Comment:
It might make sense to start alphabetizing these. I think the list is only going to get longer.